src/main.rs

```diff
@@ -168,12 +168,21 @@ fn handle_status<C: Config>(_matches: &ArgMatches,
 }
 
 
-subcommand!(use, add_use_subcommand {
-    about("Switch to using a different hermit shell")
+subcommand!(inhabit, add_inhabit_subcommand {
+    about("Switch to inhabiting a different hermit shell");
+    arg(Arg::with_name("SHELL_NAME").help("The name of the shell to inhabit"))
 });
 
-fn handle_use<C: Config>(_matches: &ArgMatches,
-                         _hermit: &mut Hermit<C>,
-                         _file_operations: &mut FileOperations) {
-    println!("hermit use is not implemented yet.")
+fn handle_inhabit<C: Config>(matches: &ArgMatches,
+                         hermit: &mut Hermit<C>,
+                         file_operations: &mut FileOperations) {
+    let shell_name = matches.value_of("SHELL_NAME").unwrap_or("");
+    hermit.inhabit_shell(file_operations, shell_name);
+    let mut shell_names = Vec::new();
```

Comment:
you don't need this right? since you `new` up a `Vec` inside of `config.get_shell_list()` this is just being overwritten.